Tests/GRDBTests/FoundationNSDecimalNumberTests.swift

```diff
@@ -3,8 +3,10 @@ import GRDB
 
 class FoundationNSDecimalNumberTests: GRDBTestCase {
 
-    func testNSDecimalNumberPreservesIntegerValues() {
-        
+    func testNSDecimalNumberPreservesIntegerValues() throws {
+        #if !canImport(ObjectiveC)
+        throw XCTSkip("NSDecimalNumber unavailable")
```

Comment:
NSDecimalNumber is part of the [GRDB support for decimals](https://github.com/groue/GRDB.swift/blob/master/README.md#nsnumber-nsdecimalnumber-and-decimal).

I find mentions of NSDecimalNumber in https://github.com/swiftlang/swift-corelibs-foundation and https://github.com/swiftlang/swift-foundation.

Is this lack of NSDecimalNumber temporary, or Android specific? Shouldn't we use `#if os(android)` instead of `#if !canImport(ObjectiveC)`?

Finally, shouldn't we disable the whole test class, since it is named Foundation**NSDecimalNumber**Tests?